scope_ascom.cpp

```diff
@@ -136,6 +136,12 @@ wxArrayString ScopeASCOM::EnumAscomScopes()
     return list;
 }
 
+static bool AlreadyPresentInGit(GITEntry deviceEntry)

```

Comment:
`GITEntry` is not expected to be passed by value. If it is not done properly we can have temporary intermediate values that invoke ~GITEntry and that would be really bad.
Passing by reference would be much better here.  It can be const reference since Get() is a const method.
Also, let's keep the same capitalization convention referring to the GIT in all caps in class names & function names:
```suggestion
static bool AlreadyPresentInGIT(const GITEntry& deviceEntry)
```
The other approach (which I would prefer) would be to make this a member function of the GITEntry class:
```
class GITEntry
{
...
    bool IsRegistered() const
    {
        ...
    }
};
```